src/inmanta/server/services/orchestrationservice.py

```diff
@@ -936,6 +930,19 @@ async def release_version(
                     connection=connection,
                 )
 
+        increments: tuple[abc.Set[ResourceIdStr], abc.Set[ResourceIdStr]] = await self.resource_service._get_increment(
+            env, version_id
+        )
+
+        increment_ids, neg_increment = increments
+        await self.resource_service._mark_deployed(env, neg_increment, now, lambda x: True, version_id)
```

Comment:
I think we should remove the underscores from these method names, because they belong to the public API of the class while the name suggests otherwise.